main.js

```diff
@@ -0,0 +1,80 @@
+const open = require('open');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers')
+const { fs } = require('appium-support');
```

Comment:
this module is not supported anymore. Consider switching to `@appium/support` one